pytorch_lightning/trainer/trainer.py

```diff
@@ -137,7 +137,7 @@ def __init__(
         tpu_cores: Optional[Union[List[int], str, int]] = None,
         ipus: Optional[int] = None,
         log_gpu_memory: Optional[str] = None,
-        progress_bar_refresh_rate: Optional[int] = None,
+        progress_bar_refresh_rate: Optional[int] = None,  # TODO: remove in v1.7
```

Comment:
will we do a follow up PR for changing all our tests that use this argument, to simplify the removal in 1.7?

Reply:
ah yes I will do that when I add the `enable_progress_bar` flag